int16 sys_byte_num
---
bool recieved